src/app/homepage/MediaSection.tsx

```diff
@@ -3,24 +3,39 @@ import { FontVariant } from '@/app/theme'
 import styled from '@emotion/styled'
 import { SectionHeader } from '@/components/Section'
 import React from 'react'
-import { Section, GridContainer } from './Styles'
+import { Section } from './Styles'
 
 const MediaArea = styled.div`
   display: flex;
   flex-direction: row;
   flex: auto;
   flex-wrap: wrap;
   justify-content: space-between;
-  gap: 72px;
+  gap: 32px;
 `
+
+const VideoCard = styled.div`
+  flex: 1 1 0;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  justify-content: normal;
+`
+const VideoContainer = styled.div`
+  max-height: 300px;
+  aspect-ratio: 16 / 9;
+`
+
 const VideoTitle = styled.h3`
   ${FontVariant.title_md}
   text-align: center;
 `
+
 const VideoDate = styled.h4`
   text-align: center;
   font-size: 16px;
 `
+
 const videos = [
```

Comment:
can we move `videos` to Data folder? Make it similar to `Members`.

Reply:
https://github.com/kixlab/website/pull/21/commits/5c143cd5995b289269560874f37ce70c97881864